samples/go/nomdex/nomdex_command.go

```diff
@@ -0,0 +1,68 @@
+// Copyright 2016 Attic Labs, Inc. All rights reserved.
+// Licensed under the Apache License, version 2.0:
+// http://www.apache.org/licenses/LICENSE-2.0
+
+package main
+
+import (
+	"fmt"
+	"os"
+	"strings"
+
+	flag "github.com/tsuru/gnuflag"
+)
+
+type nomdexCommand struct {
```

Comment:
Much of this infrastructure for subcommands is shared with the `noms` command -- Did you consider factoring it out?